### ISSUE TITLE
torch script result not same as pytorch result when i modify on the default cfg

### ISSUE THREAD

**Issue by AhmedHessuin** [facebookresearch/detectron2#3639]
## Instructions To Reproduce the 🐛 Bug:
1. Full runnable code or full changes you made:
i just changed the anchors size and aspect ratio in the cfg
```
cfg.MODEL.ANCHOR_GENERATOR.ASPECT_RATIOS = [[0.25, 0.5, 1.,2,2.5]]
cfg.MODEL.ANCHOR_GENERATOR.SIZES= [[8], [16], [32], [64], [128]]
```
If making changes to the project itself, please use output of the following command:
git rev-parse HEAD; git diff
<put code or diff here>
2. What exact command you run:
```
/home/appuser/detectron2_repo/tools/deploy/export_model.py --config-file cfg.yaml    --output ./output --export-method tracing --format torchscript --sample-image input.jpg    MODEL.WEIGHTS model_final.pth MODEL.DEVICE cpu
```
3. __Full logs__ or other relevant observations:
```
```
4. please simplify the steps as much as possible so they do not require additional resources to
   run, such as a private dataset.
i trained model normally with default anchor boxes sizes and ratio once, and with modified aspect ratio and anchor boxes sizes once
## Expected behavior:
when transferring the pytorch to torch script with default anchors size and ratio i get the same result 
when transferring the pytorch to torch script with modified anchors size and ratio i get a different results, not 1% change but new result 

expected to get the same result when changing the anchors size, noted that i changed it in the cfg.yaml given to export model

## Environment:
using the docker image in ```https://github.com/facebookresearch/detectron2/tree/master/docker```
Provide your environment information using the following command:
```
wget -nc -q https://github.com/facebookresearch/detectron2/raw/master/detectron2/utils/collect_env.py && python collect_env.py
```

**Comment by ppwwyyxx** [comment]
> expected to get the same result when changing the anchors size, noted that i changed it in the cfg.yaml given to export model


After changing the anchors, the models need to be retrained to work at all.
And, because anchors are changed, the model is different and it is not expected to produce the same results any more. 
Closing because the expectation is not correct.

**Comment by AhmedHessuin** [comment]
@ppwwyyxx  
i changed the anchors then trained the model. 
and i got let's say 90% accuracy 

when i deploy the model with the changed anchors ( same as trained) i get  accuracy 80 %
